package.json

```diff
@@ -29,7 +33,7 @@
     "morgan": "~1.9.1",
     "parcel": "^1.12.4",
     "prettier": "^1.19.1",
-    "react": "^16.12.0",
+    "react": "^16.8.0 || ^17.0.0",
```

Comment:
What is the reason for this version constraint?

Reply:
Taken care in upcoming PR